lib/cssLoadChecker/README.md

```diff
@@ -0,0 +1,19 @@
+# CSS Load Checker
+
+This module provides a means of checking if a CSS file has been loaded already or not. It does this by adding a test div to the page and checking the computed styles of it for the `display: none` style.
+
+```js
+var cssLoadChecker = require('bv-ui-core/lib/cssLoadChecker');
+
+if (cssLoadChecker.checkForCss('bv-verify-css-loaded')) {
```

Comment:
Promises would require a library, and we're trying to keep to zero dependencies here.